src/EnergyPlus/FluidProperties.hh

```diff
@@ -342,11 +343,11 @@ namespace FluidProperties {
     struct cached_tsh
     {
         // Members
-        Int64 iT;
+        std::uint64_t iT;
         Real64 sh;
 
         // Default Constructor
-        cached_tsh() : iT(-1000), sh(0.0)
+        cached_tsh() : iT(1000), sh(0.0)
```

Comment:
I haven't grokked yet why this went from -1000 to 1000 initialization, but I'll probably figure it out as I keep reviewing.

Reply:
It's because it went from an ```Int64``` to ```std::uint64_t```